src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -40,10 +40,20 @@ foam.CLASS({
         args = {
           ...args,
           indicateProcessing: true,
-          borderColor: this.theme.grey2,
-          borderColorHover: this.theme.grey2,
+          borderColor: 'rgba(0,0,0,0)',
+          borderColorHover: 'rgba(0,0,0,0)',
           label: '' + number
         };
+      } else if ( wizardlet.indicator == this.WizardletIndicator.NETWORK_FAILURE ) {
+        args = {
+          ...args,
+          borderColor: this.theme.destructive2,
+          borderColorHover: this.theme.destructive2,
+          backgroundColor: this.theme.destructive2,
+          icon: this.theme.glyphs.networkError.getDataUrl({
+            fill: this.theme.white
+          })
+        }
```

Comment:
Can we remove the network indicator from the Wizardlet section. I feel like the banner would be enough and the circle isn't big enough for an icon

Reply:
The icon could be made smaller, but I'm not sure how to do that yet. I think it's good to have it here because it indicates to the user which sections are inconsistent with the stored information.

Reply:
I don't think the user will know what that means. Especially since there is nothing they can do about it. But if we do want to keep it then we should definitely make it smaller, maybe just replace it with the loading spinner on the circle indicator.

Reply:
Replacing with the loading spinner seems like a good option, 'cause they just need to know their input isn't saved.

Reply:
If two of 12 items had a red indicator - with a tooltip, it would be clear which sections are not saved, and the user can decide their worth and wether to wait or abandon their input.

Reply:
I'm fine with that. No border on the circle indicator while the loading spinner is active.

Reply:
@jlhughes We already use the spinner as a way of indicating unsaved data. Adding more unnecessary icons would only make it more confusing